ORLib/build.gradle

```diff
@@ -2,8 +2,12 @@ plugins {
     id 'com.android.library'
     id 'kotlin-android'
     id 'org.jetbrains.kotlin.android'
+    id 'maven-publish'
+    id 'signing'
 }
 
+version = hasProperty("orlibVersion") ? orlibVersion : "1.2.0-SNAPSHOT"
+
```

Comment:
Same comment here about version

Reply:
I've updated the PR so it now uses [axion-release-plugin](https://github.com/allegro/axion-release-plugin/) for determining the version numbers.